packages/web-components/src/radio/radio.styles.ts

```diff
@@ -0,0 +1,173 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '@microsoft/fast-foundation';
+import {
+  borderRadiusMedium,
+  colorCompoundBrandForeground1,
+  colorCompoundBrandForeground1Pressed,
+  colorCompoundBrandStroke,
+  colorCompoundBrandStrokeHover,
+  colorCompoundBrandStrokePressed,
+  colorNeutralForeground1,
+  colorNeutralForeground2,
+  colorNeutralForeground3,
+  colorNeutralForegroundDisabled,
+  colorNeutralStrokeAccessible,
+  colorNeutralStrokeAccessibleHover,
+  colorNeutralStrokeAccessiblePressed,
+  colorStrokeFocus1,
+  colorStrokeFocus2,
+  fontFamilyBase,
+  fontSizeBase300,
+  fontWeightRegular,
+  lineHeightBase300,
+  spacingHorizontalS,
+  spacingHorizontalXS,
+  spacingVerticalS,
+  spacingVerticalXS,
+} from '../theme/design-tokens.js';
+
+/** Radio styles
+ * @public
+ */
+export const styles = css`
+  ${display('inline-flex')}
+
+  :host {
+    align-items: center;
+    flex-direction: row;
```

Comment:
@brianchristopherbrady if you use the slotted selector you can likely just swap the direction to column when radio-group is stacked and adjust any other flex properties.

A note on this scenario - while not necessary _now_ if we support namespacing (changing `fluent-` to `dcs-` we likely will want to use the `tagFor()` API to get the correct tag for the class instead of hardcoding it). I don't know that we need to enable that _now_ but just a heads up.